internal/database/database_test.go

```diff
@@ -296,43 +412,48 @@ func TestGetFiles(t *testing.T) {
 			FileID:                    "file1",
 			DatasetID:                 "dataset1",
 			DisplayFileName:           "file.txt",
+			FilePath:                  "dir/file.txt",
 			FileName:                  "urn:file1",
 			FileSize:                  60,
 			DecryptedFileSize:         32,
 			DecryptedFileChecksum:     "hash",
 			DecryptedFileChecksumType: "sha256",
 			Status:                    "ready",
+			CreatedAt:                 "a while ago",
+			LastModified:              "now",
 		}
 		expected = append(expected, fileInfo)
 		query := `
 			SELECT files.stable_id AS id,
 				datasets.stable_id AS dataset_id,
 				reverse\(split_part\(reverse\(files.submission_file_path::text\), '/'::text, 1\)\) AS display_file_name,
+				files.submission_file_path AS file_path,
 				files.archive_file_path AS file_name,
 				files.archive_file_size AS file_size,
 				files.decrypted_file_size,
 				sha.checksum AS decrypted_file_checksum,
 				sha.type AS decrypted_file_checksum_type,
-				log.event AS status
+				log.event AS status,
+				files.created_at,
+				files.last_modified
 			FROM sda.files
 			JOIN sda.file_dataset ON file_id = files.id
 			JOIN sda.datasets ON file_dataset.dataset_id = datasets.id
 			LEFT JOIN \(SELECT file_id, \(ARRAY_AGG\(event ORDER BY started_at DESC\)\)\[1\] AS event FROM sda.file_event_log GROUP BY file_id\) log ON files.id = log.file_id
 			LEFT JOIN \(SELECT file_id, checksum, type FROM sda.checksums WHERE source = 'UNENCRYPTED'\) sha ON files.id = sha.file_id
 			WHERE datasets.stable_id = \$1;
 		`
-
-		mock.ExpectQuery(strings.ReplaceAll(strings.ReplaceAll(query, "\t", ""), "\n", " ")).
+		mock.ExpectQuery(query).
 			WithArgs("dataset1").
 			WillReturnRows(sqlmock.NewRows([]string{"file_id", "dataset_id",
-				"display_file_name", "file_name", "file_size",
+				"display_file_name", "file_path", "file_name", "file_size",
 				"decrypted_file_size", "decrypted_file_checksum",
-				"decrypted_file_checksum_type", "file_status"}).AddRow(
-				fileInfo.FileID, fileInfo.DatasetID,
-				fileInfo.DisplayFileName, fileInfo.FileName,
+				"decrypted_file_checksum_type", "file_status", "created_at",
+				"last_modified"}).AddRow(fileInfo.FileID, fileInfo.DatasetID,
+				fileInfo.DisplayFileName, fileInfo.FilePath, fileInfo.FileName,
 				fileInfo.FileSize, fileInfo.DecryptedFileSize,
 				fileInfo.DecryptedFileChecksum, fileInfo.DecryptedFileChecksumType,
-				fileInfo.Status))
+				fileInfo.Status, fileInfo.CreatedAt, fileInfo.LastModified))
```

Comment:
If you want to break the long lines from the queries but still have it readable it should probably look more like this:
```
		mock.ExpectQuery(query).
			WithArgs("dataset1").
			WillReturnRows(sqlmock.NewRows([]string{
				"file_id",
				"dataset_id",
				"display_file_name",
				"file_path",
				"file_name",
				"file_size",
				"decrypted_file_size",
				"decrypted_file_checksum",
				"decrypted_file_checksum_type",
				"file_status",
				"created_at",
				"last_modified"},
			).AddRow(
				fileInfo.FileID, fileInfo.DatasetID,
				fileInfo.DisplayFileName,
				fileInfo.FilePath,
				fileInfo.FileName,
				fileInfo.FileSize,
				fileInfo.DecryptedFileSize,
				fileInfo.DecryptedFileChecksum,
				fileInfo.DecryptedFileChecksumType,
				fileInfo.Status,
				fileInfo.CreatedAt,
				fileInfo.LastModified,
			))
```